### PR TITLE
small change in grammar to understand clear

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/admissionregistration/validation/validation_test.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func TestValidateInitializerConfiguration(t *testing.T) {
 			}
 		} else {
 			if test.expectedError != "" {
-				t.Errorf("test case %s, unexpected no error, expected to contain %s", test.name, test.expectedError)
+				t.Errorf("test case %s, expected error contain %s", test.name, test.expectedError)
```
Comment (rootfs) [inline pkg/apis/admissionregistration/validation/validation_test.go]: contains (same below)

**Comment by zjj2wry** [comment]
@rootfs done ~

**Comment by NickrenREN** [comment]
@k8s-bot ok to test

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-e2e-kops-aws test this

**Pull request closed without merging**